### PR TITLE
Use Azure Open AI package to generate cases for model

### PR DESCRIPTION
**Pull request by haritha-mohan**
Ran into some interesting time limit cases:

> Requests to the ChatCompletions_Create Operation under Azure OpenAI API version 2024-05-01-preview have exceeded call rate limit of your current OpenAI S0 pricing tier. Please retry after 10 seconds. Please go here: https://aka.ms/oai/quotaincrease if you would like to further increase the default rate limit.

so limited to one call to the API per invocation of the tool.

### CODE REVIEW

File: comments/classified.csv [hunk 1]
```diff
@@ -3203,6 +3208,7 @@ Good Luck,0,0.5
 Good vibes,0,0.5
 gotcha - will fix both here and in XCT soon,0,0.5
 Great idea,0,0.5
+Great job on including detailed comments throughout your code, making it easy to understand.,0,0.5
```
Comment (jonathanpeppers) [inline comments/classified.csv:3211]: Ok, this line has a `,`:
```suggestion
"Great job on including detailed comments throughout your code, making it easy to understand.",0,0.5
```

File: comments/classified.csv [hunk 2]
```diff
@@ -4320,6 +4338,8 @@ The first change adds CNS_INT long 0 Fseq[_00] for source field in the field by
 The first THREE were rejected out of hand and the following half a dozen where similarly ditched. I could produce ANOTHER fukkin half a dozen but it that wouldn't matter to a pure PoS paid scumbag would it,1,0.5
 The fix above was merged,0,0.5
 THE GOD THAT YOU HATE HATES YOU,1,0.5
+The implementation generally follows the project's coding guidelines.,0,0.5
+The initial logic is clear, but refactoring could make it even clearer.,0,0.5
```
Comment (haritha-mohan) [inline comments/classified.csv:4342]:
```suggestion
"The initial logic is clear, but refactoring could make it even clearer.",0,0.5
```

File: comments/classified.csv [hunk 3]
```diff
@@ -4357,6 +4377,7 @@ The text for #code shouldn't be deleted,0,0.5
 The thread was aborted,0,0.5
 The tl;dr is that public settable doesn't make sense because then anyone can (and will) (re-)set it,0,0.5
 The total regression across the 200K methods is 269 bytes. Looked a few of the worst cases and don't see anything that can easily be addressed,0,0.5
+The variable names you chose are very intuitive, enhancing the readability of your code.,0,0.5
```
Comment (haritha-mohan) [inline comments/classified.csv:4380]:
```suggestion
"The variable names you chose are very intuitive, enhancing the readability of your code.",0,0.5
```

File: comments/classified.csv [hunk 4]
```diff
@@ -4434,6 +4455,7 @@ This feels a little gross to me,1,0.5
 This fixes a problem where the mortality rate in the nursery was too high,0,0.5
 This fixes a problem,0,0.5
 This fixes the layout of child controls,0,0.5
+This function performs as expected, but there is room for code simplification.,0,0.5
```
Comment (haritha-mohan) [inline comments/classified.csv:4458]:
```suggestion
"This function performs as expected, but there is room for code simplification.",0,0.5
```

File: comments/classified.csv [hunk 5]
```diff
@@ -5083,10 +5110,12 @@ Your are a filthy fuck,1,0.5
 YOUR ARE FUCKING GAY WAD,1,0.5
 Your code Is a pain to read,1,0.5
 Your code is completely wrong,1,0.5
+Your code is well-structured and follows best practices, making it easy for others to contribute.,0,0.5
```
Comment (haritha-mohan) [inline comments/classified.csv:5113]:
```suggestion
"Your code is well-structured and follows best practices, making it easy for others to contribute.",0,0.5
```

File: comments/classified.csv [hunk 6]
```diff
@@ -4301,14 +4316,17 @@ The BigInteger type is an immutable type that represents an arbitrarily large in
 The bounds (128 => 512) can be tightened further here in that case. I will extract these to consts,0,0.5
 The built-in platforms also use color for interactive controls,0,0.5
 The code has been reviewed and confirmed to be correct. Great work! @github,0,0.5
+The code passes all the tests, which is good. Additional tests might be useful.,0,0.5
```
Comment (haritha-mohan) [inline comments/classified.csv:4319]:
```suggestion
"The code passes all the tests, which is good. Additional tests might be useful.",0,0.5
```

**Review by jonathanpeppers — APPROVED** [review]

**Pull request merged**